client/secureclient/src/main/java/com/linkedin/openhouse/client/ssl/WebClientFactory.java

```diff
@@ -61,13 +65,17 @@ protected WebClient createWebClient(
       @NonNull String baseUrl, String token, String truststoreLocation)
       throws MalformedURLException, SSLException {
     String transportProtocol = getTransportProtocol(baseUrl);
+    HttpClient httpClient = null;
     if (HTTPS.equals(transportProtocol)) {
-      return createSecureWebClient(baseUrl, truststoreLocation);
+      httpClient = createSecureHttpClient(truststoreLocation);
```

Comment:
small refactor to deduplicate very similar methods

instead of building WebClient, just get HttpClient and pass to shared method for getting webClient